compiler/packages/snap/src/SproutTodoFilter.ts

```diff
@@ -498,6 +498,7 @@ const skipFilter = new Set([
   "useState-pruned-dependency-change-detect",
   "useState-unpruned-dependency",
   "useState-and-other-hook-unpruned-dependency",
+  "change-detect-reassign",
```

Comment:
oh interesting, what was the error for this fixture?

Reply:
It uses the react-compiler-runtime, IIUC that's not available in fixtures right?